routers/actions.js

```diff
@@ -0,0 +1,142 @@
+const express = require('express');
+
+const db = require('../data/helpers/actionModel');
+const dbProjects = require('../data/helpers/projectModel');
+
+const router = express.Router();
+
+// MIDDLEWARE
+
+const stripReqBody = (req, res, next) => {
+  const newReqBody = {};
+  const allowed = ['project_id', 'description', 'notes', 'completed'];
+  Object.keys(req.body).forEach(key => {
+    if(allowed.includes(key)) {
+      newReqBody[key] = req.body[key]
+    }
+  });
+  req.body = newReqBody;
+  next();
+} // Strips all properties that won't be sent to DB
+
+const hasProjectID = (req, res, next) => {
```

Comment:
great job using your middlewares! I would pull them out into a different file and include them later for readability though!